src/renderers/dom/shared/ReactDOMDebugTool.js

```diff
@@ -62,6 +62,27 @@ var ReactDOMDebugTool = {
   onTestEvent() {
     emitEvent('onTestEvent');
   },
+  onBeginProcessingChildContext() {
+    emitEvent('onBeginProcessingChildContext');
+  },
+  onEndProcessingChildContext() {
+    emitEvent('onEndProcessingChildContext');
+  },
+  onSetState() {
+    emitEvent('onSetState');
+  },
+  onMountRootComponent(internalInstance) {
```

Comment:
Now that #6549 landed, the argument is `debugID` rather than `internalInstance`.
Would you mind rebasing on that?